packages/devkit/src/generators/generate-files.ts

```diff
@@ -1,6 +1,6 @@
 import { readFileSync, readdirSync, statSync } from 'fs';
 import * as path from 'path';
-import type { Tree } from 'nx/src/config/tree';
```

Comment:
We could put a TS file called `nx-reexports.ts` or something in packages/devkit/src to avoid all of these deep imports.... I'm not sure that its worth changing **_NOW_**, but it may be good to do eventually to limit the places they happen.